crates/factor-variables/build.rs

```diff
@@ -0,0 +1,6 @@
+fn main() {
+    println!("cargo::rerun-if-changed=build.rs");
```

Comment:
`cargo`-culted from https://github.com/bytecodealliance/wasmtime/blob/0e5048e20a24f6da19b3794eeb94111f052dc414/crates/component-macro/build.rs#L2

From the cargo docs:
> If the build script inherently does not need to re-run under any circumstance, then emitting cargo::rerun-if-changed=build.rs is a simple way to prevent it from being re-run (otherwise, the default if no rerun-if instructions are emitted is to scan the entire package directory for changes).